epevermodbus/driver.py

```diff
@@ -69,7 +93,7 @@ def get_load_power(self):
 
     def get_battery_current(self):
         """Battery current in amps"""
-        return self.retriable_read_long(0x331B, 4) / 100
```

Comment:
good catch 👍

Reply:
I'm glad you ... signed off on this one ;)